tests/unit/commands/migration/test_migration.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+# Copyright 2007-2017 UShareSoft SAS, All rights reserved
+#
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import unittest
+
+import pyxb
+from mock import patch
+from uforge.application import Api
+from uforge.objects import uforge
+
+from hammr.commands.migration import migration
+from hammr.utils import constants
+
+
+class MyTestCase(unittest.TestCase):
+    @patch('uforge.application.Api._Users._Migrations.Getall')
+    @patch('texttable.Texttable.add_row')
+    def test_do_list_list_all_the_migrations(self, mock_table_add_row, mock_api_getall):
+        # given
+        m = migration.Migration()
+        m.api = Api("url", username="username", password="password", headers=None,
+                    disable_ssl_certificate_validation=False, timeout=constants.HTTP_TIMEOUT)
+        m.login = "login"
+        m.password = "password"
+        self.create_migration(mock_api_getall, 1, "a migration", 50, False)
```

Comment:
We should test with more than 1 migration, what do you think?

Reply:
agree, I will add a second migration that is complete because the code is not the same in this case

Reply:
I am agree